### ISSUE TITLE
rooms_unused_list global crashes: unregistered_route

### ISSUE THREAD

**Issue by marzzzello** [processone/ejabberd#3726]
## Environment

- ejabberd version: 21.12
- Erlang version: `10.5`
- OS: Linux (Ubuntu)
- Installed from: official docker image

## Errors:

`ejabberdctl rooms_unused_list global 10`

```
Failed RPC connection to the node ejabberd@70b4d508a990: {'EXIT',
                                                          {undef,
                                                           [{lib,
                                                             format_exception,
                                                             [2,error,
                                                              {unregistered_route,
                                                               <<"global">>},
                                                              [{ejabberd_router,
                                                                host_of_route,
                                                                1,
                                                                [{file,
                                                                  "src/ejabberd_router.erl"},
                                                                 {line,280}]},
                                                               {mod_muc_admin,
                                                                get_all_rooms,
                                                                1,
                                                                [{file,
                                                                  "src/mod_muc_admin.erl"},
                                                                 {line,864}]},
```

## Bug description
The command `rooms_unused_list` with global as service produces an error.

The command `ejabberdctl rooms_unused_list conference.domain.tld 10` works fine

**Comment by badlop** [comment]
I can reproduce the problem with ejabberd compiled from source code, and older versions. The bug appeared in 2e8023158d068413101639a4e91f53de08282b0f which was first included in ejabberd 20.12, a year ago. Apparently few people use that option, or that command.

I'll check in the next days for a solution.

**Comment by badlop** [comment]
That recent commit fixes the problem.
